src/PKSim.Infrastructure/ProjectConverter/v11/Converter10to11.cs

```diff
@@ -186,6 +186,13 @@ private void addExpressionProfilesUsedBySimulationSubjectToProject(ISimulationSu
             expressionProfile.Category = simulationSubject.Name;
             _expressionProfileUpdater.SynchronizeExpressionProfileWithSimulationSubject(expressionProfile, simulationSubject);
 
+            //Some parameters are probably marked as FixedValue event thought they have not changed (Formula=>constant) due to change in 
+            //definition of Fraction expressed basolateral going from a constant to a formula. We reset
+            expressionProfile.Individual.AllMoleculeParametersFor(expressionProfile.Molecule)
```

Comment:
This is gbecause we explictely overwrite formula with constant in this very specific case (the comment says for Permeability formula)...